### PR TITLE
Add merge_test_matrix script to generate a test matrix file

### PR DESCRIPTION
**Pull request by clairlin53**
This script could be used to automate the process of generating test matrix from initial test results. It helps QA to have the test scope for projects.

Jira card: https://warthogs.atlassian.net/browse/CQT-2849

### CODE REVIEW

File: Tools/PC/merge_test_matrix/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Run the Merge test matrix script
+
+## Purpose
+
+This tool is designed to generate a test matrix file by hardware componets.
+
+## Usage
+
+1. place the tool and tarball files obtained from initial tests into the same folder.
```
Comment (baconYao) [inline Tools/PC/merge_test_matrix/README.md:9]: Capital the first alphabet

File: Tools/PC/merge_test_matrix/merge_test_matrix.py
Comment (baconYao) [inline Tools/PC/merge_test_matrix/merge_test_matrix.py:1]: Looks like this script is migrated from https://git.launchpad.net/oem-qa-tools/tree/merge_test_matrix.py.
May I know is there any logical change of this script?

**Review by baconYao — CHANGES_REQUESTED** [review]
I don't think we need the `sample_test_matrix.xlsx` in repo, instead, we need some testing tarballs as golden testing data.

I suggest that we need a test_data directory which is used to store all the test tarballs.

**Comment by clairlin53** [comment]
> I don't think we need the `sample_test_matrix.xlsx` in repo, instead, we need some testing tarballs as golden testing data.
> 
> I suggest that we need a test_data directory which is used to store all the test tarballs.

Tarbell file from the initial test result is based on the project, in my mind it's no meaning to put it here.

**Comment by baconYao** [comment]
> > I don't think we need the `sample_test_matrix.xlsx` in repo, instead, we need some testing tarballs as golden testing data.
> > I suggest that we need a test_data directory which is used to store all the test tarballs.
> 
> Tarbell file from the initial test result is based on the project, in my mind it's no meaning to put it here.

Agree.
After discussing with Clair, there's no need to put any files such as sample xlsx or testing tarball in this tool.

**Review by stanley31huang — APPROVED** [review]
LGTM

**Review by scotthu27 — APPROVED** [review]
LGTM

**Review by baconYao — APPROVED** [review]
LGTM

**Pull request merged**